device_tags_python/src/app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask, render_template
+app = Flask(__name__, template_folder='../views', static_folder='../views/public')
+
+import os
+import datetime
+from balena import Balena
+balena = Balena({
```

Comment:
I would call it something else e.g. balena_sdk to avoid conficliting with the existing toplevel name `balena` from the library